src/Translatable/Translatable.php

```diff
@@ -162,6 +162,10 @@ public function getAttribute($key)
                 return;
             }
 
+            if ($this->hasGetMutator($key)) {
+                return $this->mutateAttribute($key, $this->getTranslation($locale)->$key);
```

Comment:
Why do you call call `$this->mutateAttribute()` instead of `$this->getAttributeValue($key)`?